tests/local/test_PacerNefEmailTest.py

```diff
@@ -0,0 +1,67 @@
+import fnmatch
+import os
+import unittest
+
+from juriscraper.pacer.email import NefEmail
+from tests import TESTS_ROOT_EXAMPLES_PACER
+
+TESTS_ROOT_EXAMPLES_PACER_NEF = os.path.join(TESTS_ROOT_EXAMPLES_PACER, "nef")
+
+
+class PacerNefEmailTest(unittest.TestCase):
+    def run_parsers_on_path(
```

Comment:
@mlissner To your first comment. This method checks the fields on all the notification email files, and I took this pattern from some of the other test suites. After running this, I proceeded t some selective individual verification below. Since we only had two examples for Notification Emails, it was not too overkill. What this means is that:

1. When new files are added, they will be automatically run through this generic test method.
2. They don't "necessarily" require the individual level testing that these first two received.

If you prefer I just remove the individual level tests because it would set an unsustainable precedent, that is fine. I just wanted to describe what this was doing.

Reply:
Got it, yeah, I'd much prefer the way `tests.local.test_PacerParseAppellateDocketTest` works, because it catches **any** changes in the parser, in any field, and it makes my life as a reviewer much easier b/c I can see the output that is being generated and don't have to try to deduce it by reading the code. Do you mind? I think if you just subclass `PacerParseTestCase`, you should get it up and running pretty easily.

Reply:
No problem. I can model after that, I just took one of the patterns and ran with it. But that one is fine.